source/common/http/conn_manager_config.h

```diff
@@ -63,6 +63,10 @@ namespace Http {
   COUNTER(downstream_rq_max_duration_reached)                                                      \
   COUNTER(downstream_rq_ws_on_non_ws_route)                                                        \
   COUNTER(rs_too_large)                                                                            \
+  COUNTER(downstream_buffer_overflow_total)                                                        \
+  COUNTER(upstream_buffer_overflow_total)                                                          \
+  COUNTER(filter_buffer_overflow_total)                                                            \
+  COUNTER(response_buffer_overflow_total)                                                          \
```

Comment:
Per my other comments it might be worth it to try to list out the various stats we want to have and discuss somewhere?